Extensions/3D/JsExtension.js

```diff
@@ -2216,8 +2219,14 @@ module.exports = {
           this._centerY / objectTextureFrame.height;
 
         this._pixiTexturedObject.angle = this._instance.getAngle();
-        this._pixiTexturedObject.scale.x = width / objectTextureFrame.width;
-        this._pixiTexturedObject.scale.y = height / objectTextureFrame.height;
+        const scaleX =
+          (width / objectTextureFrame.width) *
+          (this._instance.isFlippedX() ? -1 : 1);
```

Comment:
I'm wondering if all those `(this._instance.isFlippedX() ? -1 : 1)` can be replaced with just a `Math.abs()`